integrations/tensorflow/test/iree_tf_tests/math/vulkan__dynamic_dim_mod.run

```diff
@@ -1,4 +1,4 @@
 # REQUIRES: vulkan
 # REQUIRES: bugfix
-#           TODO(#8083): re-enable once passing
+#           Fails on NVIDIA Turing
```

Comment:
Hmm, probably relates to that in Vulkan "For the OpSRem and OpSMod instructions, if either operand is negative the result is undefined." I'll take a look later.

Reply:
Filed https://github.com/iree-org/iree/issues/9670 to track this.